js/sdk/src/components/EmailCapture.js

```diff
@@ -57,51 +52,91 @@ const EmailCaptureSchema = Yup.object().shape(
   [['soundcloud', 'twitter', 'instagram']]
 )
 
-const EmailCapture = ({ size }) => {
+const EmailCapture = ({ setChildFormOpen, setParentOpen }) => {
   const { wallet } = useContext(Wallet.Context)
+  const { enqueueSnackbar } = useSnackbar()
+  const [usingMagicWallet, setUsingMagicWallet] = useState(false)
+  const [user, setUser] = useState(undefined)
   const { publicKey, connected } = wallet
-  const { submitEmailRequest } = useContext(Nina.Context)
+  const { submitEmailRequest, getVerificationsForUser, verificationState } =
+    useContext(Nina.Context)
   const [open, setOpen] = useState(false)
   const [showSuccessInfo, setShowSuccessInfo] = useState(false)
-  const handleOpen = () => setOpen(true)
-  const handleClose = () => setOpen(false)
   const [formValues, setFormValues] = useState({})
   const [formIsValid, setFormIsValid] = useState(false)
   const [submitButtonText, setSubmitButtonText] = useState('Submit')
-
-  useEffect(() => {
-    if (open) {
-      logEvent('email_request_initiated', 'engagement')
-    }
-  }, [open])
+  const [pending, setPending] = useState(false)
+  const [userVerifications, setUserVerifications] = useState(undefined)
+  const [soundcloudAccount, setSoundcloudAccount] = useState(undefined)
+  const [twitterAccount, settwitterAccount] = useState(undefined)
```

Comment:
Very minor small nitpick thing here, but we should rename this to setTwitterAccount to maintain consistent camelcase.